src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -60,36 +60,36 @@ typedef enum {
 } EnOssanCutsceneState;
 
 typedef enum {
-    /* 00 */ FSN_ANIMATION_IDLE,
-    /* 01 */ FSN_ANIMATION_SCRATCH_BACK,
-    /* 02 */ FSN_ANIMATION_TURN_AROUND_FORWARD,
-    /* 03 */ FSN_ANIMATION_TURN_AROUND_REVERSE,
-    /* 04 */ FSN_ANIMATION_HANDS_ON_COUNTER_START,
-    /* 05 */ FSN_ANIMATION_HANDS_ON_COUNTER_LOOP,
-    /* 06 */ FSN_ANIMATION_HAND_ON_FACE_START,
-    /* 07 */ FSN_ANIMATION_HAND_ON_FACE_LOOP,
-    /* 08 */ FSN_ANIMATION_LEAN_FORWARD_START,
-    /* 09 */ FSN_ANIMATION_LEAN_FORWARD_LOOP,
-    /* 10 */ FSN_ANIMATION_SLAM_COUNTER_START,
-    /* 11 */ FSN_ANIMATION_SLAM_COUNTER_LOOP,
-    /* 12 */ FSN_ANIMATION_MAKE_OFFER,
-    /* 13 */ FSN_ANIMATION_MAX
+    /* 00 */ FSN_ANIM_IDLE,
```

Comment:
Can you remove the leading 0 in the offset comments, and just use spaces to keep them aligned.